src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -110,7 +110,7 @@ public JVillagerTradeProfession(MCreator mcreator, IHelpContext gui, JPanel pare
 	}
 
 	public void reloadDataLists() {
-		ComboBoxUtil.updateComboBoxContents(villagerProfession, ElementUtil.loadAllVillagerProfessions());
+		ComboBoxUtil.updateComboBoxContents(villagerProfession, ElementUtil.loadAllVillagerProfessions(workspace));
```

Comment:
You can use `mcreator`'s workspace here as well as on line 60.
```suggestion
		ComboBoxUtil.updateComboBoxContents(villagerProfession, ElementUtil.loadAllVillagerProfessions(mcreator.getWorkspace()));
```

Reply:
Line 60 doesn't use `workspace`

Reply:
Done!